### ISSUE TITLE
adding new version with yarn installs winston@3

### ISSUE THREAD

**Issue by abimelex** [figo-connect/node-figo#54]
I guess this is not the expected behaviour since winston in code looks still like version 2 is used:

Version 3.1.1 installs still the old winston version 👍 

```
$ yarn add figo@3.1.1
yarn add v1.12.3
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Saved lockfile.
success Saved 7 new dependencies.
info Direct dependencies
└─ figo@3.1.1
info All dependencies
├─ ....
└─ winston@2.4.4
✨  Done in 3.00s.
```

Version 3.1.2 installs the new winston 3 version . 👎 

```
$ yarn add figo@3.1.2
yarn add v1.12.3
[1/4] 🔍  Resolving packages...
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
[4/4] 📃  Building fresh packages...
success Saved lockfile.
success Saved 21 new dependencies.
info Direct dependencies
└─ figo@3.1.2
info All dependencies
├─ ....
├─ winston-transport@4.3.0
└─ winston@3.2.1
✨  Done in 2.58s.
```

**Comment by christianhuening** [comment]
so we need to rollback to Winston 2.4.4 ?

**Comment by christianhuening** [comment]
@abimelex Please try again, should be solved with 3.1.3

**Comment by christianhuening** [comment]
ok seemingly does not work with Winston 2.4.4 since something breaks. We will have to investigate

**Comment by abimelex** [comment]
But 3.1.1. will work with the current API?
Why is there actually no major version if the functionality with the new API breaks?

**Comment by abimelex** [comment]
fixed with 3.1.4
